packages/components/reorder/tests/reorder.test.tsx

```diff
@@ -192,7 +192,9 @@ describe("<Reorder />", () => {
       },
     ])
 
-    expect(onChange).toHaveBeenCalledWith(["Item 2", "Item 1"])
-    expect(onCompleteChange).toHaveBeenCalledWith(["Item 2", "Item 1"])
+    await waitFor(() => {
+      expect(onChange).toHaveBeenCalledWith(["Item 2", "Item 1"])
+      expect(onCompleteChange).toHaveBeenCalledWith(["Item 2", "Item 1"])
+    })
   })
```

Comment:
Please fix it like this.
Only one assertion may be contained within `waitFor`.
```tsx
await waitFor(() => expect(onChange).toHaveBeenCalledWith(["Item 2", "Item 1"]))
expect(onCompleteChange).toHaveBeenCalledWith(["Item 2", "Item 1"])
```

Reply:
Fixed.